app/contact/page.tsx

```diff
@@ -0,0 +1,233 @@
+"use client";
+import React, { useState, useEffect } from "react";
+import Lottie from "react-lottie";
+import animationData from "./mail.json";
+import Link from "next/link";
+import { IoMail } from "react-icons/io5";
+import { FaDiscord, FaLinkedin, FaTwitter, FaNetworkWired } from "react-icons/fa";
+import { BallTriangle } from "react-loading-icons";
+import { handleInputChange, handleSubmit } from "./formUtils";
+
+const AirtableForm = () => {
+  const initialFormData = {
```

Comment:
Please use `types` when you are using Typescript else there is no objective to using Typescript.